kratos/utilities/embedded_skin_utility.h

```diff
@@ -277,13 +350,74 @@ class KRATOS_API(KRATOS_CORE) EmbeddedSkinUtility
         const Condition::NodesArrayType &rNewNodesArray);
 
     /**
-     * @brief Set the Skin Entities Properties 
-     * This method checks which is the last properties id. and 
+     * @brief Sets a pointer to a new skin condition
+     * From the split pattern of an intersected element, this method
+     * creates and returns a pointer to a new skin condition.
+     * @param rOriginGeometryType GeometryType of the condition to be created
+     * @param rNewNodesArray array containing pointers to the nodes that will
+     * conform the condition
+     * @param rConditionId new condition identifier
+     * @param pConditionProperties pointer to the new condition properties
+     * @return Condition::Pointer pointer to a new skin condition
+     */
+    Condition::Pointer SetNewConditionPointer(
```

Comment:
create?

Reply:
Renamed to `pCreateNewCondition`. Same has been done with the `pCreateNewConditionGeometry`.